### PR TITLE
feat: delete favorite activities function created

### PR DESCRIPTION
**Pull request by ArthurEncr**
# Description

Allow users to remove items from my favorites activities list

Jira ticket: https://apps.nrs.gov.bc.ca/int/jira/browse/FSADT2-322

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have already been accepted and merged

### CODE REVIEW

File: src/components/Card/index.tsx [hunk 1]
```diff
@@ -23,8 +25,7 @@ const Card = ({
         <Icon className="card-icon" />
         <OverflowMenu className="card-overflow" ariaLabel="overflow-menu">
           <OverflowMenuItem tabIndex="0" itemText={highlighted ? 'Dehighlight shortcut' : 'Highlight shortcut'} onClick={highlightFunction} />
-          <OverflowMenuItem itemText="Item 2" />
-          <OverflowMenuItem itemText="Item 3" />
+          <OverflowMenuItem itemText="Delete shortcut" onClick={deleteFunction} />
```
Comment (mgaseta) [inline src/components/Card/index.tsx:28]: Change the text here, "Remove Favorite" reflects better the function and it is more user friendly :)

File: src/components/Card/index.tsx [hunk 2]
```diff
@@ -11,10 +11,12 @@ interface CardProps {
   description: string;
   highlighted?: boolean;
   highlightFunction?: () => void;
+  deleteFunction?: () => void;
+
```
Comment (mgaseta) [inline src/components/Card/index.tsx:15]: Remove empty line.

**Review by mgaseta — CHANGES_REQUESTED** [review]
Just 2 small markups, great job overall! 🥇 🚀

File: src/components/Card/index.tsx (hunk 1 above)
Comment (ArthurEncr) [reply src/components/Card/index.tsx:28]: The text was based on figma/jira, it's a change we can suggest to the UX team

File: src/components/Card/index.tsx (hunk 2 above)
Comment (ArthurEncr) [reply src/components/Card/index.tsx]: done

**Review by mgaseta — APPROVED** [review]
LGTM!

**Pull request merged**